importApp/src/test/java/com/commercetools/dataimport/orders/OrdersImportJobConfigurationIntegrationTest.java

```diff
@@ -0,0 +1,148 @@
+
+
+        final OrderQuery orderQuery = OrderQuery.of().withLimit(0);
+        final PagedQueryResult<Order> categoryPagedQueryResult = sphereClient.executeBlocking(orderQuery);
+
+        assertThat(categoryPagedQueryResult.getTotal()).isEqualTo(ordersCountUsingEmailAndOrderId(csvPath));
+
+        OrderQuery query = OrderQuery.of().withPredicates(model -> model.customerEmail().is("303"))
+                .plusPredicates(model -> model.orderNumber().is("1"));
+        final PagedQueryResult<Order> firstOrderResult = sphereClient.executeBlocking(query);
+
+        assertThat(firstOrderResult.getCount()).isEqualTo(1);
+        Order order = firstOrderResult.getResults().get(0);
+
+        assertThat(order.getTotalPrice()).isEqualTo(MoneyImpl.ofCents(73375, "EUR"));
+
+        assertThat(order.getLineItems()).extracting("variant")
+                .extracting("sku")
+                .containsExactly("A0E20000000252G", "M0E20000000DZKB", "A0E200000002AD2");
+
+        assertThat(order.getLineItems()).extracting("quantity")
+                .containsExactly(1L, 1L, 1L);
+
+    }
+
+
+    public static Product createProductWithSkus(final BlockingSphereClient sphereClient, Set<String> skus) {
+        PriceDraft priceDraft = PriceDraft.of(MoneyImpl.of(new BigDecimal("123456"), "EUR")).withCountry(CountryCode.DE);
+        List<ProductVariantDraft> productVariantDrafts = skus.stream().map(sku -> ProductVariantDraftBuilder.of().sku(sku).build()).collect(Collectors.toList());
+
+        final ProductType productType = createProductType(sphereClient);
+        final ProductDraftBuilder productDraftBuilder = ProductDraftBuilder.of(productType, LocalizedString.of(Locale.ENGLISH, "product-name"),
+                LocalizedString.of(Locale.ENGLISH, RandomStringUtils.randomAlphabetic(10)), productVariantDrafts);
+        final Product product = sphereClient.executeBlocking(ProductCreateCommand.of(productDraftBuilder.publish(true).build()));
+        final AddPrice addPrice = AddPrice.of(product.getMasterData().getCurrent().getMasterVariant().getId(), priceDraft);
+        Product productWithPrice = sphereClient.executeBlocking(ProductUpdateCommand.of(product, asList(addPrice, Publish.of())));
+        return productWithPrice;
+    }
+
+    private static List<String> extractSkusFromCSV(Path csvPath) throws IOException {
+        return Files.newBufferedReader(csvPath)
+                .lines()
+                .skip(1)
+                .filter(str -> !str.isEmpty())
+                .map(line -> line.split(",")[2].trim())
+                .collect(Collectors.toList());
+    }
+
+    private static long ordersCountUsingEmailAndOrderId(Path csvPath) throws IOException {
+        return Files.newBufferedReader(csvPath)
+                .lines()
+                .skip(1)
+                .filter(str -> !str.isEmpty())
+                .map(line -> line.split(",")[0].trim() + "_" + line.split(",")[1].trim())
+                .distinct()
```

Comment:
👍  for 2nd and 3rd line. Unfortunately 1st line cannot be represented as method reference due to the `!`.
It would be cool if they'd add an `isNotEmpty` method, as nowadays having both positive and negative boolean methods really help readability 🤔